_book/02-first-steps/05-npm-usage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Usage with npm"
+---
+
+The latest stable version is available via NPM registry. The version distributed via NPM it the WebAssembly build. It can be installed with: 
+
+```bash
+npm install verovio
+```
+
+## Basic usage
```

Comment:
This is basic usage for what? Node environments? Given that the vast majority of people will use the JS version of Verovio in a browser environment, should we prioritize that as "Basic usage" first?